java/src/main/java/com/microsoft/graph/bulk/sample/UploadWithBulkLoader.java

```diff
@@ -0,0 +1,105 @@
+import com.azure.cosmos.CosmosAsyncContainer;
+import com.azure.cosmos.CosmosAsyncDatabase;
+import com.azure.cosmos.CosmosClientBuilder;
+import com.azure.cosmos.models.CosmosContainerProperties;
+import com.azure.cosmos.models.CosmosContainerRequestOptions;
+import com.azure.cosmos.models.CosmosContainerResponse;
+import com.azure.cosmos.models.CosmosDatabaseResponse;
+import com.azure.cosmos.models.ThroughputProperties;
+import com.microsoft.graph.bulk.impl.BulkGremlinObjectMapper;
+import com.microsoft.graph.bulk.impl.impl.CosmosDBSQLBulkExecutor;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import reactor.core.publisher.Mono;
+
+@Slf4j
+public class UploadWithBulkLoader<V, E> {
+    private final CosmosAsyncClient client;
+
+    private CosmosAsyncDatabase database;
+    private CosmosAsyncContainer container;
+
+    private final CosmosDBSQLBulkExecutor<V, E> executor;
+
+    @SneakyThrows
+    public UploadWithBulkLoader() {
+        client = new CosmosClientBuilder()
+                .endpoint(DatabaseSettings.HOST)
+                .key(DatabaseSettings.MASTER_KEY)
+                .contentResponseOnWriteEnabled(true)
+                .consistencyLevel(ConsistencyLevel.SESSION)
+                .buildAsyncClient();
+
+        createDatabaseIfNotExists();
+        createContainerIfNotExists();
+
+        executor = new CosmosDBSQLBulkExecutor<>(
+                container, BulkGremlinObjectMapper.getBulkGremlinObjectMapper(),
+                DatabaseSettings.ALLOW_UPSERT);
+    }
+
+    private void createDatabaseIfNotExists() throws Exception {
+        log.info("Create database " + DatabaseSettings.DATABASE_NAME + " if not exists.");
+
+        //  Create database if not exists
+        //  <CreateDatabaseIfNotExists>
+        Mono<CosmosDatabaseResponse> databaseIfNotExists = client.createDatabaseIfNotExists(DatabaseSettings.DATABASE_NAME);
+        databaseIfNotExists.flatMap(databaseResponse -> {
+            database = client.getDatabase(databaseResponse.getProperties().getId());
+            log.info("Checking database " + database.getId() + " completed!\n");
+            return Mono.empty();
+        }).block();
+        //  </CreateDatabaseIfNotExists>
+    }
+
+    private void createContainerIfNotExists() throws Exception {
+        log.info("Create container " + DatabaseSettings.CONTAINER_NAME + " if not exists.");
+
+        //  Create container if not exists
+        //  <CreateContainerIfNotExists>
+
+        CosmosContainerProperties containerProperties = new CosmosContainerProperties(
+                DatabaseSettings.CONTAINER_NAME, DatabaseSettings.PARTITION_KEY_PATH);
+        ThroughputProperties throughputProperties = ThroughputProperties.createManualThroughput(DatabaseSettings.THROUGHPUT);
+        Mono<CosmosContainerResponse> containerIfNotExists = database.createContainerIfNotExists(containerProperties, throughputProperties);
+
+        //  Create container with 400 RU/s
```

Comment:
nit: the comment may not correct. The container will be created with the configured throughput, so may not be 400RU?